### DISCUSSION TITLE
Manual or documents on harmonic balance method

### DISCUSSION THREAD

**Discussion by jjywow** [su2code/SU2#1942] [Q&A]
Hello,

I read some papers using the harmonic balance method implemented in SU2.

I'm hoping I can predict flutter using the harmonic balance method in SU2.

Where can I find the manual or documents for the harmonic balance method?

I like to know what functions or options are available for the method. But, tutorials are mostly about time-domain solvers. 

Thank you for helping!

**Comment by bigfooted** [comment]
This implementation and turbomachinery capabilities in general are a bit scattered over different branches unfortunately. Some work is being done to move some stuff into develop.
@NAnand-TUD Who is working on this at the moment at TU Delft? Maybe you can give an update?

**Comment by NAnand-TUD** [comment]
Hello,

Indeed, this branch is way way behind in time. The code is in Antonio Rubino's git (a SU2 veteran) (https://github.com/arubino/SU2/tree/feature_MZHB). Maybe have a look there first and then I can reply to any specific questions you might have. 

I am not aware of anyone using HB right not at TU Delft. However, it will be best to send a message to @matteopini to stay in loop.

Thanks,
N Anand

**Reply by jjywow** [reply]
Hello. Thanks for the comments. I wonder how I can determine whether the flutter occurs or not for the given shape and aerodynamic conditions using the harmonic balance method. Does the Harmonic Balance method provide me with the eigenvalues so that I can determine whether flutter occurs or not? Or can turbomachinery do that?
